components/brave_new_tab_ui/components/search/SearchBox.tsx

```diff
@@ -48,7 +48,7 @@ const Container = styled.div`
 
 export default function SearchBox() {
   const { searchEngine, query, setQuery } = useSearchContext()
-  const placeholderText = searchEngine?.host === braveSearchHost
+  const placeholderText = searchEngine?.host === defaultSearchHost
```

Comment:
Hmm not sure if this is right - this is meant to show "Search the web privately" for Brave and "Search" for non-Brave engines.

Reply:
oh, thanks for catching! Fixed.